app/src/main/java/xyz/whereuat/whereuat/ui/views/KeyLocDialogFragment.java

```diff
@@ -85,6 +93,13 @@ public void onFocusChange(View v, boolean hasFocus) {
         });
     }
 
+    /**
+     * This function wraps an insert into the key location table.
+     *
+     * @param context the calling context
```

Comment:
Do we want to capitalize `Context` here? It conflicts with line 41 of AutoShareStar.java
Should probably do it one way or the other